trade-finance-manager-ui/server/controllers/utilisation-reports/index.ts

```diff
@@ -1,27 +1,47 @@
 import { Request, Response } from 'express';
+import { subMonths } from 'date-fns';
 import api from '../../api';
 import { getFormattedReportDueDate, getFormattedReportPeriod } from '../../services/utilisation-report-service';
 import { getIsoMonth } from '../../helpers/date';
 import { getReportReconciliationSummaryViewModel } from './helpers';
 import { asString } from '../../helpers/validation';
 
+export const getUtilisationReportsSummaryData = async (userToken: unknown) => {
+  const currentDate = new Date();
+  const submissionDate = subMonths(currentDate, 1);
+  const oneIndexedSubmissionMonth = submissionDate.getMonth() + 1;
+  const submissionYear = submissionDate.getFullYear();
```

Comment:
As discussed, these should refer to the start of the report period, instead of submission.